spec/fixtures/constants.rb

```diff
@@ -25,6 +25,8 @@
 SAMPLE_ANALYZE_TARGET = "=== ANALYZE TARGET The Spacer OF PROJECT Pods WITH THE DEFAULT CONFIGURATION Debug ==="
 SAMPLE_CLEAN = "=== CLEAN TARGET Pods-ObjectiveSugar OF PROJECT Pods WITH CONFIGURATION Debug ==="
 SAMPLE_ANOTHER_CLEAN = "=== CLEAN TARGET Pods OF PROJECT Pods WITH CONFIGURATION Debug ==="
+SAMPLE_BUILD_SUCCEEDED = "** BUILD SUCCEEDED **"
+SAMPLE_CLEAN_SUCCEEDED = "** CLEAN SUCCEEDED **"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.